### PR TITLE
VR-13293 utf8 with mssql

### PR DESCRIPTION
**Pull request by evgeniilezhnin**
## Impact and Context

utf8 support in mssql

## How to Revert

revert the PR

### CODE REVIEW

**Comment by evgeniilezhnin** [comment]
need first make sure that this is supported. UTF-8 is supported since SQL Server 2019

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java [hunk 1]
```diff
@@ -166,13 +166,20 @@ private String getDatasourceClass(RdbConfig rdbConfiguration) {
...
-              "ALTER DATABASE `%s` CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;", dbName);
+      String sql;
+      if (rdbConfiguration.isMssql()) {
+        sql =
+            String.format(
+                "ALTER DATABASE \"%s\" COLLATE Latin1_General_100_CI_AI_SC_UTF8;", dbName);
+      } else {
+        sql =
+            String.format(
+                "ALTER DATABASE `%s` CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;", dbName);
```
Comment (eheinlein-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java:181]: My understanding is that `utf8mb4_unicode_ci` is better to use than `utf8mb4_general_ci`.  I know this code was already using `utf8mb4_general_ci`, and making this change now might cause problems, but I wanted to make a note.

Comment (evgeniilezhnin) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java:181]: makes sence, but the current change was about utf-8 in Microsoft SQL, this line is about mysql

**Comment by evgeniilezhnin** [comment]
will be using utf-16 instead utf-8

**Pull request closed without merging**